### DISCUSSION TITLE
Title already exists and no new media types found

### DISCUSSION THREAD

**Discussion by bruor** [seerr-team/seerr#519] [Q&A]
Most of my content is not marked as available even though it exists in Emby. 
The scan is seeing the media but not updating it's availability
![image](https://github.com/Fallenbagel/jellyseerr/assets/454516/a8e95aeb-7344-4f79-88d1-60208eb7de84)

**Comment by fallenbagel** [comment]
I just linked you to discord on another discussion. I think I'm pretty sure what your issue exactly is but I want to make sure and run you through some troubleshooting steps so hop on discord do it's easier to follow througha

**Reply by dnielso5** [reply]
was there an answer for this, i get it as well.

**Reply by fallenbagel** [reply]
> was there an answer for this, i get it as well.

Please hop onto discord for troubleshooting as this might be dependent on your setup.

**Reply by zerolin1010** [reply]
> 有没有答案，我也明白了。

Has your problem been solved?
